test/system/helpers.network.bash

```diff
@@ -449,3 +449,11 @@ function default_addr() {
     local expr='[.[0].addr_info[] | select(.deprecated != true)][0].local'
     ip -j -"${ip_ver}" addr show "${ifname}" | jq -rM "${expr}"
 }
+
+function all_addr() {
+    local ip_ver="${1}"
+    local ifname="${2:-$(default_ifname "${ip_ver}")}"
+
+    local expr='[.[0].addr_info[] | select(.deprecated != true)]'
+    ip -j -"${ip_ver}" addr show "${ifname}" | jq -rM "${expr}" | grep local | cut -d'"' -f4 | tr '\n' ' '
```

Comment:
Why all this grep, cur, tr thing? If we only want the local addresses we can just add them to the jq filter.
```
'[.[0].addr_info[] | select(.deprecated != true)].[].local'
```
which seems to return the same result unless I am missing something